src/fetch_payment_token_request.cc

```diff
@@ -0,0 +1,33 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#include "fetch_payment_token_request.h"
+#include "ads_serve_helper.h"
+
+#include "base/logging.h"
+
+namespace confirmations {
+
+FetchPaymentTokenRequest::FetchPaymentTokenRequest() = default;
```

Comment:
`PaymentTokenRequestBuilder` or just `PaymentTokenRequest`

Reply:
FetchPaymentTokenRequest is due to the server API naming the task FetchPaymentToken

Reply:
I see, but the convention in chromium is still to use `*Builder` for stuff like this

Reply:
it seems a little unnecessary to even instantiate this in its current form since all the state is transient. I think we either want a `Builder` pattern with `set_confirmation_id` and maybe `set_endpoint_version` (default to 1) or just make them static and dump the class